.github/workflows/iotics-identity.yml

```diff
@@ -43,13 +43,13 @@ jobs:
     needs: validate
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Python 3.7
+      - name: Set up Python 3.8
         uses: actions/setup-python@v2
         with:
-          python-version: '3.7'
+          python-version: '3.8'
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip wheel==0.36.2
+          python -m pip install --upgrade pip wheel==0.40.0
```

Comment:
Why specific version of `wheel==0.40.0`, instead of the latest one (which is `0.41.0`) like the `pip`